gst/nnstreamer/tensor_aggregator/tensor_aggregator.c

```diff
@@ -124,15 +124,15 @@ enum
 static GstStaticPadTemplate sink_template = GST_STATIC_PAD_TEMPLATE ("sink",
     GST_PAD_SINK,
     GST_PAD_ALWAYS,
-    GST_STATIC_CAPS (GST_TENSOR_CAP_DEFAULT));
+    GST_STATIC_CAPS (GST_TENSORS_CAP_WITH_NUM ("1")));
```

Comment:
for backward compatibility, support both other/tensor and other/tensors,num=1
```
GST_STATIC_CAPS (CAPS_STRING GST_TENSOR_CAP_DEFAULT ";" GST_TENSORS_CAP_WITH_NUM ("1")));
```

Reply:
Definitely right. Fixed!!